scripts/test-backwards.sh

```diff
@@ -17,12 +17,8 @@ setup(){
     }
     random_bytes_file 2x1024      "$TEST_FILES_DIR/small-upload-testfile" # create 2kb file of random bytes (inline)
     random_bytes_file 5x1024x1024 "$TEST_FILES_DIR/big-upload-testfile"   # create 5mb file of random bytes (remote)
-    echo "setup test successfully"
-}
 
-cleanup(){
-    rm -rf "$STORJ_NETWORK_DIR"
```

Comment:
why remove this?

Reply:
Because the outside test is deleting the directory and the download is done twice. Once with the new uplink and once with the last release uplink.

Reply:
Also, the cleanup is called explicitly as the last thing using the last release uplink.

Reply:
makes sense.